### PR TITLE
Modify policy on vswitch delete and port forward

### PR DESCRIPTION
**Pull request by XiaowenJiang**
1. will not delete openvswitch if there are still ports on that vswitch.
2. move packet forwarding things from vswitch to port_forwarding section.

Signed-off-by: Julie Jiang <julie.jiang@emc.com>

### CODE REVIEW

**Comment by bcyan** [comment]
@weia5-emc Please help us to review the changes

File: ivn/core.py [hunk 1]
```diff
@@ -670,35 +669,42 @@ def __preinit(self, io_interfaces):
...
-
-    @staticmethod
-    def build(portforward, logger):
-        rules = portforward.get("rules", None)
-        io_interfaces = portforward.get("io_interfaces", None)
-        if rules and io_interfaces:
-            worker = InfrasimPortforward(logger)
-            worker.__preinit(io_interfaces)
-            for rule in rules:
+    def __build_rules(self):
```
Comment (weia5-emc) [inline ivn/core.py:672]: 👍

**Review by weia5-emc — APPROVED** [review]
🉑

**Comment by weia5-emc** [comment]
image

**Pull request merged**